### ISSUE TITLE
About Split the training set?

### ISSUE THREAD

**Issue by AramSo** [qianguih/voxelnet#23]
In Data Preparation,
"Split the training set into training and validation set according to the protocol here. And rearrange the folders to have the following structure"

 I don't know exactly what this means.

- I ran crop.py. So, Files in the velodyne folder have been removed prefix (000). Why? Is not overwrite ?
- Do you mean to manually relocate the img / point file in the training / validation folder in the order of the txt file you attached?

**Comment by lkhazyf** [comment]
I have the same problem.

**Comment by JustWon** [comment]
I also wondered how the training is going..
as you mentioned, I downloaded the ImageSets.tar.gz file and checked the text files in it.
and acoording to the text file (train.txt, val.txt), I reorganized the structure of the dataset
and the training is running.

but still, I don't know what the crop.py is doing.

**Comment by bjtuchenxu** [comment]
> I have the same problem.

你好  你这个代码跑出来了吗？？可以添加个联系方式讨论一下吗？

**Comment by bjtuchenxu** [comment]
> I also wondered how the training is going..
> as you mentioned, I downloaded the ImageSets.tar.gz file and checked the text files in it.
> and acoording to the text file (train.txt, val.txt), I reorganized the structure of the dataset
> and the training is running.
> 
> but still, I don't know what the crop.py is doing.

the crop.py is used to crop th raw data

**Comment by GydaRagnarsdottir** [comment]
I have written a small program to do this, if you have trouble separating the data, just use this:([https://github.com/GydaRagnarsdottir/KITTIDataSeperater])
